joomla/joomla-cms
# Missing space in Mod custom

Pull Request for Issue # .
#### Summary of Changes

added the missing space between 'custom' and 'suffix'
#### Testing Instructions

create a custom module, look at the module class and you'll see "custom"
add a class suffix and you'll see "customYOURSUFFIX"  this is not as expected, nor consistent with the rest of the module class suffix behaviors.
after this patch you'll see that it is now "custom YOURSUFFIX"
